app/Http/Controllers/CardDAV/CardDAVController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers\CardDAV;
+
+use App\Http\Controllers\Controller;
+
+// https://github.com/sabre-io/dav/blob/master/tests/Sabre/DAVACL/PrincipalBackend/AbstractPDOTest.php
+// https://github.com/sabre-io/dav/blob/master/lib/DAVACL/PrincipalBackend/PDO.php
+// https://github.com/sabre-io/dav/tree/master/lib/DAV/Browser
+// https://github.com/sabre-io/dav/blob/master/lib/CardDAV/Backend/PDO.php
+// http://sabre.io/dav/caldav-carddav-integration-guide/
+
+use Illuminate\Support\Facades\Auth;
+
+class CardDAVController extends Controller
+{
+    /**
+     * Display the specified resource.
+     */
+    public function init()
+    {
+        // Disable debugger for caldav output
+        \Debugbar::disable();
+
+        // TODO: Not sure if this is needed, check later
+        date_default_timezone_set('Europe/Berlin');
```

Comment:
If anything, probably should be set to UTC?

Reply:
Agreed